misc/scripts/vfile-swarm-config.txt

```diff
@@ -0,0 +1,7 @@
+5
```

Comment:
how about following instead of just values?
NODE_COUNT=5
MGR_COUNT=3

It increases readability and user will not forget to supply mandatory values.

Reply:
Done.

Reply:
Seems the sample config file is not updated to add NODE_COUNT and MGR_COUNT :)

Reply:
Fix it in the  sample config file.